index.js

```diff
@@ -4,19 +4,21 @@ module.exports = TinySDF;
 
 var INF = 1e20;
 
-function TinySDF(fontSize, buffer, radius, cutoff, fontFamily) {
+function TinySDF(fontSize, buffer, radius, cutoff, fontFamily, fontWeight) {
     this.fontSize = fontSize || 24;
     this.buffer = buffer === undefined ? 3 : buffer;
     this.cutoff = cutoff || 0.25;
     this.fontFamily = fontFamily || 'sans-serif';
+    this.fontWeight = fontWeight || 'normal';
     this.radius = radius || 8;
     var size = this.size = this.fontSize + this.buffer * 2;
 
     this.canvas = document.createElement('canvas');
     this.canvas.width = this.canvas.height = size;
 
     this.ctx = this.canvas.getContext('2d');
-    this.ctx.font = fontSize + 'px ' + this.fontFamily;
+    /* style | variant | weight | size/line-height | family */
+    this.ctx.font = `normal normal ${this.fontWeight} ${fontSize}px/normal ${this.fontFamily}`;
```

Comment:
I switched to the format 'font-weight font-size font-family'. I read the "Shorthand font property: the 'font' property" section of the CSS spec, and it seems to me to be a little ambiguous about how it distinguishes 'font-weight'/'font-variant'/'font-style'. But I guess it can disambiguate based on the value (except for 'normal' vs 'inherited').